examples/miscellaneous/plot_outlier_detection_bench.py

```diff
@@ -186,8 +186,14 @@ def compute_prediction(X, model_name):
             name=model_name,
             linewidth=linewidth,
             ax=axs[i // cols, i % cols],
+            plot_chance_level=True,
+            chance_level_kwargs={
+                "linewidth": linewidth,
+                "linestyle": ":",
+                "color": "g",
```

Comment:
I think we can let to the default color (black is fine). If you really want green, we need to use `tab10:green` to have the right green color.